robotd/visiond/camera_base.py

```diff
@@ -0,0 +1,21 @@
+
+class CameraBase:
+    def __init__(self):
+        self.initialised = False
+        self.cam_image_size = None
+
+    def init(self):
+        """ Initialise the camera"""
+        self.initialised = True
+
+    def get_image_size(self):
+        if not self.initialised:
+            raise Exception("Must Initialise camera before getting image size")
```

Comment:
Generally a code smell to raise `Exception`: I suspect `RuntimeError` would be better here.

Also, you might consider mentioning `.init()` here in the error message to reduce the work someone has to do if they see this error.